packages/eslint-plugin/docs/rules/no-floating-promises.md

```diff
@@ -73,6 +75,33 @@ void returnsPromise();
 void Promise.reject('value');
 ```
 
+### `ignoreIIFE`
+
+This allows to skip checking of async iife
+
+Examples of **correct** code for this rule with `{ ignoreIIFE: true }`:
```

Comment:
```suggestion
Examples of **correct** code for this rule with `{ ignoreIIFE: false }`:
```

Reply:
I have kept `ignoreIIFE` false by default. 
so we need to add docs example when it is `true`, right !

Reply:
yeah sure. Don't need the false example.
Though I would avoid splitting the code block in two

eg do

---

Examples of **correct** code for this rule with `{ ignoreIIFE: true }`:

```ts

<examples>

```

<<end section>>

Reply:
didnt get this .

You meant like this ?

**Before**
![image](https://user-images.githubusercontent.com/26347874/77852902-b45f5580-71fe-11ea-9e42-fa8923f9968f.png)


**After**
![image](https://user-images.githubusercontent.com/26347874/77852881-998ce100-71fe-11ea-9f3c-24742f150ed5.png)